syft/cataloger/ruby/parse_gemspec.go

```diff
@@ -0,0 +1,133 @@
+package ruby
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io"
+	"regexp"
+	"strings"
+
+	"github.com/mitchellh/mapstructure"
+
+	"github.com/anchore/syft/syft/cataloger/common"
+	"github.com/anchore/syft/syft/pkg"
+)
+
+// integrity check
+var _ common.ParserFn = parseGemFileLockEntries
+
+type postProcessor func(string) []string
+
+var patterns = map[string]*regexp.Regexp{
```

Comment:
I believe all instances of `{1}` in these regex patterns can be safely removed (since I don't think they're needed)